ChangeLog.md

```diff
@@ -61,6 +62,7 @@ Changes listed here are checked in to GitHub ("master" branch unless specificall
 * Docs: Fix many documentation issues, improve docs generally.
 * Docs: Expand documentation significantly, and document the new serial features.
 * Unenhancement: Remove getAnalogSampleDuration(), as I could not make anything compile with that in and had worked on it over a day.
+* Enhancement: 2 MHz is now a supported clock speed.
```

Comment:
How could I have forgotten this?! That reminds me of a few other things I may have forgotten...